### PR TITLE
Migrate Sqs Tool Window off ToolkitToolWindowManager

### PR DESCRIPTION
**Pull request by abrooksv**
* Fix coroutines not stopping on tab being closed
* Fix Tab being deleted/created again in order to give it focus
* Remove coroutines when it didn't make sense
* Fix using an odd JButton for the Send button
* Fix odd padding in the send tab

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SendMessagePane.form [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <form xmlns="http://www.intellij.com/uidesigner/form/" version="1" bind-to-class="software.aws.toolkits.jetbrains.services.sqs.toolwindow.SendMessagePane">
-  <grid id="27dc6" binding="component" layout-manager="GridLayoutManager" row-count="2" column-count="8" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
-    <margin top="10" left="10" bottom="10" right="10"/>
+  <grid id="27dc6" binding="component" layout-manager="GridLayoutManager" row-count="2" column-count="5" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
```
Comment (manodnyab) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SendMessagePane.form:3]: Can a screenshot be attached for the UI changes? Also, if the margin values are all 0s and assuming that's the default, maybe we can remove the tag unless they are generated along with the form?

**Review by manodnyab — APPROVED** [review]

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SendMessagePane.form (hunk 1 above)
Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SendMessagePane.form:3]: It is generated by the GUI editor.

Screenshot:
Screen Shot 2021-06-29 at 9 39 57 AM

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/PollMessagePane.kt [hunk 2]
```diff
@@ -63,15 +67,15 @@ class PollMessagePane(
...
-        launch {
+        coroutineScope.launch {
             getAvailableMessages()
         }
         addActionsToTable()
     }
 
     suspend fun requestMessages() {
         try {
-            withContext(Dispatchers.IO) {
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/PollMessagePane.kt:74]: should we ban `Dispatchers.*`?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SqsWindow.kt [hunk 3]
```diff
@@ -3,76 +3,71 @@
...
-            LOG.error(e) { "Exception thrown while trying to open queue '${queue.queueName}'" }
-            throw e
+
+        val toolWindow = SqsWindowFactory.getToolWindow(project)
+        val contentManager = toolWindow.contentManager
+
+        val sqsViewContent = findQueue(queue.queueUrl) ?: createSqsView(contentManager, queue)
+
+        toolWindow.show()
+        contentManager.setSelectedContent(sqsViewContent, true)
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SqsWindow.kt:37]: does set/show order matter? and does this need to be on edt?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SqsWindow.kt [hunk 4]
```diff
@@ -3,76 +3,71 @@
...
 
-    fun closeQueue(queueUrl: String) = runBlocking {
-        withContext(edtContext) {
-            toolWindow.find(queueUrl)?.dispose()
+    private fun createSqsView(contentManager: ContentManager, queue: Queue): Content {
+        val sqsView = SqsWindowUi(project, client, queue)
+        val sqsViewContent = contentManager.factory.createContent(sqsView.component, queue.queueName, false)
+        sqsViewContent.isCloseable = true
+        sqsViewContent.isPinnable = true
+        contentManager.addContent(sqsViewContent)
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SqsWindow.kt:57]: addContent definitely has an EDT assertion

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/PollMessagePane.kt (hunk 2 above)
Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/PollMessagePane.kt:74]: Ya probably

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SqsWindow.kt (hunk 3 above)
Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SqsWindow.kt:37]: It is same order I saw in IJ code, so I don't think so...its either fast enough or it happens in the same paint call?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SqsWindow.kt (hunk 4 above)
Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SqsWindow.kt:57]: Ya, all our call sites our on EDT, but will add asserts to force that

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/PollMessagePane.kt (hunk 2 above)
Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/PollMessagePane.kt:74]: #2688

**Review by rli — APPROVED** [review]

**Pull request merged**